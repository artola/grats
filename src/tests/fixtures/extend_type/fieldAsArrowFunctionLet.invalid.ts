/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export let greeting = (_: SomeType): string => {
  return `Hello World`;
};
