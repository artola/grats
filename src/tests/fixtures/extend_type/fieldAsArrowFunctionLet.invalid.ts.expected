-----------------
INPUT
----------------- 
/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export let greeting = (_: SomeType): string => {
  return `Hello World`;
};

-----------------
OUTPUT
-----------------
src/tests/fixtures/extend_type/fieldAsArrowFunctionLet.invalid.ts:7:8 - error: Expected `@gqlField` arrow function to be declared as `const`.

7 export let greeting = (_: SomeType): string => {
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
8   return `Hello World`;
  ~~~~~~~~~~~~~~~~~~~~~~~
9 };
  ~
